xml/ha_config_basics.xml

```diff
@@ -200,7 +243,82 @@
     </para>
    </important>
   </sect2>
+
+  <sect2 xml:id="sec.ha.config.basics.corosync.2-node">
+   <title>&corosync; Configuration for Two-Node Clusters</title>
+   <para>
+    When using the bootstrap scripts, the &corosync; configuration contains
+    a <literal>quorum</literal> section with the following options:
+   </para>
+   <example xml:id="ex.ha.config.basics.corosync.quorum">
+    <title>Excerpt of &corosync; Configuration for a Two-Node Cluster</title>
+    <screen>quorum {
+   # Enable and configure quorum subsystem (default: off)
+   # see also corosync.conf.5 and votequorum.5
+   provider: corosync_votequorum
+   expected_votes: 2
+   two_node: 1
+}</screen>
+   </example>
+   <para>
+    Normally you do not need to change this. Do not set
+    <literal>no-quorum-policy=ignore</literal> and <literal>two_node: 1</literal>
+    together. By default, when <literal>two_node: 1</literal> is set, the
+    <literal>wait_for_all</literal> option is automatically enabled.
+   </para>
+  </sect2>
+  <sect2 xml:id="sec.ha.config.basics.corosync.n-node">
+   <title>&corosync; Configuration for N-Node Clusters</title>
+   <para>
+    As written before, we strongly recommend a two-node cluster or a N-node
+    cluster, whereas <replaceable>N</replaceable> is an odd number.
+    In regards to quorum configuration, creating a N-node cluster can be done
+    either with:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>Adding additional nodes with the <command>ha-cluster-join</command>
+     command, or</para>
+    </listitem>
+    <listitem>
+     <para>Adapting the <filename>/etc/corosync/corosync.conf</filename> manually.</para>
+    </listitem>
+   </itemizedlist>
+   <para>Configuring <filename>/etc/corosync/corosync.conf</filename> manually
+   uses typically the following content:
+   </para>
+   <example>
+    <title>Excerpt of &corosync; Configuration for a N-Node Cluster</title>
```

Comment:
As we need to mention quite a lot of options here, I would use callouts instead of continuous text to describe the individual options and their relationship to each other.

Reply:
Ok, clarified. Thanks!